index.dd

```diff
@@ -93,28 +93,36 @@ $(DIVC boxes,
                 Adam D. Ruppe's weekly newsletter. $(SPANC twid)
             )
         )
+        $(TOUR graduation-cap, Learn,
+            $(P Quick start with $(LINK2 http://tour.dlang.org,
+                Introduction to D), explore
+                $(LINK2 $(ROOT_DIR)overview.html, major features) in D,
+                read coming from $(LINK2 $(ROOT_DIR)ctod.html, C) or
```

Comment:
"read coming from C or C++" sounds off to me. I don't have a better suggestion, though.